src/fusion/car_fuser.py

```diff
@@ -52,7 +52,8 @@ def _filter_tile(self, tilecode):
         """
         ((bx_min, by_max), (bx_max, by_min)) =\
             get_bbox_from_tile_code(tilecode)
-        df = self.bgt_df.query('(x_min < @bx_max) & (x_max > @bx_min)' +
+        df = self.bgt_df.query('(bgt_name != "fietspad") &' +
```

Comment:
Misschien mooier om ipv uitsluiten in de init een set labels mee te geven die gebruikt moeten worden?

Reply:
Eens, gedaan